<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="menu" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>310</y>
       <width>521</width>
       <height>471</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <widget class="QPushButton" name="optionsButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>519</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Opcje</string>
      </property>
     </widget>
     <widget class="QPushButton" name="closeButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>320</y>
        <width>519</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Wyjd≈∫</string>
      </property>
     </widget>
     <widget class="QPushButton" name="playButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>521</width>
        <height>151</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>122344</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Zagraj</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="logo">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>70</y>
       <width>821</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="kod">
     <property name="geometry">
      <rect>
       <x>1450</x>
       <y>830</y>
       <width>421</width>
       <height>181</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="game" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1</width>
      <height>1</height>
     </rect>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>1340</x>
       <y>160</y>
       <width>561</width>
       <height>601</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>90</y>
       <width>801</width>
       <height>721</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout"/>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
